LearnosityDemo/Program.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+
+namespace LearnosityDemo
+{
+    public class Program
```

Comment:
I suggest adding `static` to the class definition. ie. `public static class Program`

Reply:
Fixed here: https://github.com/Learnosity/learnosity-sdk-asp.net/commit/2ad77f005b6fad299f2bc28e117ce41ebb451b80